Formula/tao.rb

```diff
@@ -0,0 +1,71 @@
+class Tao < Formula
+  desc "TAO, the C++ CORBA implementation"
```

Comment:
`desc` should not contain formula name.

For further checks you may run,
`$ brew audit --strict tao`